incident/models.py

```diff
@@ -0,0 +1,67 @@
+from django.db import models
+
+# Create your models here.
+
+class Incident(models.Model):
+	#HINT: Pick location from maps api
+	injuries=models.PositiveIntegerField()
+	plateNumber=models.PositiveIntegerField()
```

Comment:
The plate number can be put as a single charfield and just validate in the views that it has part of it's value as a positive integer to simplify work for the user/reporter